src/maintenance/repositories/restricted.md

```diff
@@ -0,0 +1,37 @@
+# Restricted Packages
+
+Void offers some packages that are officially maintained, but not distributed.
+These packages are marked as restricted and must be built from their
+[void-packages](https://github.com/void-linux/void-packages) template locally.
+
+Packages can be restricted from distribution by either the upstream author or
+Void. Void reserves the right to restrict distribution of any package for
+effectively any reason, massive size being the most common. The other most
+common reason is restrictive licensing that does not allow third-party
+redistribution of source or binary packages.
+
+## Building manually
+
+This feature must be enabled explicitly. Run the following in the
```

Comment:
Magic command, needs explanation.  Probably also shouldn't be here, but rather a pointer to the corresponding documentation in void-packages.

Reply:
I would like to thing that the audience could interperet a FLAG=value in a config file. I could break it down further.